app/templates/components/img-selector.hbs

```diff
@@ -0,0 +1,8 @@
+{{#if thumbnail }}
+  <img src={{thumbnail}}  class="darkroom" style="max-width: 300px;"/>
```

Comment:
Very minor- since darkroom was removed, we could remove the class here as well.

Slightly less minor: to encourage separation of concerns, could we move styles to the (s)css file, rather than specifying them in markup?